docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+name: proof-systems-github-runners
```

Comment:
What's this? I have never seen it before.

Reply:
Added here: https://github.com/o1-labs/proof-systems/pull/2570/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3

(Dunno what this is, all questions to Serhii :grin:)